templates/utilities.html

```diff
@@ -1,21 +1,51 @@
 {% extends "layout.html" %}
 
 {% block content %}
+{% if not bronze_auth %}
```

Comment:
once again put the oauth stuff into its own template

Reply:
this can be moved.

Reply:
will do cleanup alter when everything is good with this change.